website/docs/quick-start.md

```diff
@@ -8,27 +8,67 @@ After installing KAITO, you can quickly deploy a phi-3.5-mini-instruct inference
 
 - A Kubernetes cluster with KAITO installed (see [Installation](installation))
 - `kubectl` configured to access your cluster
+- **GPU nodes available in your cluster** - You have two options:
+  - **Existing GPU nodes**: If you already have GPU nodes in your cluster, you can use the preferred nodes approach (shown below)
+  - **Auto-provisioning**: Set up automatic GPU node provisioning for your cloud provider ([Azure](azure-setup) | [AWS](aws))
+
+<!-- ## Understanding Auto-Provisioning vs Instance Types
+
+KAITO supports two approaches for GPU resource management:
+
+### Preferred Nodes Approach (Recommended for existing GPU nodes)
+This approach uses node selectors and tolerations to schedule workloads on existing GPU nodes in your cluster. It's ideal when you already have GPU nodes or want to manage node provisioning separately.
+
+### Instance Types Approach (Auto-provisioning)
+This approach specifies specific cloud provider instance types (like `Standard_NC24ads_A100_v4` for Azure or `g5.4xlarge` for AWS) and relies on auto-provisioning controllers to create new nodes as needed.
+
+For more details on auto-provisioning setup, see:
+- [Azure GPU Provisioner documentation](https://learn.microsoft.com/en-us/azure/aks/gpu-cluster)
+- [AWS Karpenter documentation](https://karpenter.sh/docs/) -->
 
 ## Deploy Your First Model
 
-Let's start by deploying a phi-3.5-mini-instruct model using a workspace configuration:
+Let's start by deploying a phi-3.5-mini-instruct model on your existing GPU nodes.
+
+First get the nodes.
+
+```bash
+kubectl get nodes -l accelerator=nvidia
+```
+
+The output should look similar to this, showing all your GPU nodes.
+
+```
+NAME                                  STATUS   ROLES    AGE     VERSION
+gpunp-26695285-vmss000000             Ready    <none>   2d21h   v1.31.9
+```
+
+Now label your GPU nodes to match the label selector. We'll use the label `apps=llm-inference` for this example.
+
+```bash
+kubectl label nodes gpunp-26695285-vmss000000 apps=llm-inference
+```
+
+Create a YAML file named `phi-3.5-workspace.yaml` with the following content:
+
 
 ```yaml title="phi-3.5-workspace.yaml"
 apiVersion: kaito.sh/v1beta1
 kind: Workspace
 metadata:
   name: workspace-phi-3-5-mini
 resource:
-  instanceType: "Standard_NC24ads_A100_v4"
```

Comment:
`.resource.instanceType` is a required field, so let's not remove it

Reply:
see my comment above